### PR TITLE
Complexfield derfault value & deleted node_modules

### PR DESCRIPTION
**Pull request by Davee02**

### CODE REVIEW

File: dynamic-elements/dynamic-complexfield/dynamic-complexfield.component.ts [hunk 1]
```diff
@@ -55,17 +55,23 @@ export class TdDynamicComplexfieldComponent extends AbstractControlValueAccessor
...
     this.data_provider = dataProvider;
   }
-  
+
   ngAfterViewInit() {
     this.control = new FormControl();
   }
 
   ngOnInit(): void {
+    const defaultValue = this.control.value;
```
Comment (makuepfer) [inline dynamic-elements/dynamic-complexfield/dynamic-complexfield.component.ts:69]: i don't really get it how you handle the default value here. Needs further discussion.

Comment (makuepfer) [reply dynamic-elements/dynamic-complexfield/dynamic-complexfield.component.ts:69]: discussed via phone call.
Value in the formfield is initially set by covalent and equals the passed default value. In our case that would be the "natkey". The filter afterwards is the query function that is executed and not a filter that is applied on that value.

TL;DR; Will work that way and is ok
